utilities/tools/platform/Set-PlatformPipelinesTable.ps1

```diff
@@ -0,0 +1,132 @@
+﻿<#
+.SYNOPSIS
+Update the table of Platform pipelines in the given Readme.md
+
+.DESCRIPTION
+Update the table of Platform pipelines in the given Readme.md. The header must be identifyable with the `## Platform` header.
+
+.PARAMETER FilePath
+Mandatory. The path to the ReadMe file to update
+
+.PARAMETER RepositoryName
+Mandatory. The name of the repository the pipelines/workflows are in (required to generate the correct links)
+
+.PARAMETER Organization
+Mandatory. The name of the Organization the pipelines/workflows are in (required to generate the correct links)
+
+.PARAMETER Environment
+Mandatory. The environment to generate the badges for (either Azure DevOps - or GitHub)
+
+.PARAMETER ProjectName
+Mandatory. The name of the Azure DevOps project the pipelines are in (required to generate the correct links) - in case Azure DevOps is used
+
+.EXAMPLE
+Set-PlatformPipelinesTable -FilePath 'C:\ResourceModules\README.md' -Environment 'GitHub' -RepositoryName 'ResourceModules' -Organization 'Azure'
+
+Updated the 'README.md' file with the Platform badges from the 'https://github.com/Azure/ResourceModules' GitHub project.
+
+.EXAMPLE
+Set-PlatformPipelinesTable -FilePath 'C:\ResourceModules\README.md' -Environment 'ADO' -RepositoryName 'ResourceModules' -Organization 'carml' -ProjectName 'carml'
+
+Updated the 'README.md' file with the Platform badges from the 'https://dev.azure.com/carml/carml' Azure DevOps project.
+#>
+function Set-PlatformPipelinesTable {
```

Comment:
I'd suggest to align naming between `Set-PlatformPipelinesTable` and `Set-ReadMeModuleTable`. 
```suggestion
function Set-ReadMePlatformTable {
```

Reply:
Renamed across the board

Reply:
The ps1 file should be renamed as well, together with the references where it is imported

Reply:
Fixed